src/Raven.Studio/wwwroot/Content/css/themes/light-theme-colors-override.less

```diff
@@ -1,3 +1,23 @@
+@black: #292D3D;
+@gray-900: #484D63;
+@gray-800: #666D8A;
+@gray-700: #8085A1;
+@gray-600: #999DB7;
+@gray-500: #AFB2C6;
+@gray-400: #C5C8D4;
+@gray-300: #DBDDE3;
+@gray-200: #E5E6EA;
+@gray-100: #f6f7f7;
+@white: #ffffff;
+
+/*@gray-base: @black;
```

Comment:
remove?

Reply:
No, there is some problem with less compilation, the colors are not assigned correctly.  Hovewer this is the way I want it defined once we move to SCSS.

Reply:
ok